Makefile

```diff
@@ -11,15 +11,16 @@ all: build
 .PHONY: clean
 clean:
 	mvn clean
-	rm -f ./$(artifact_core-name).jar
+	rm -f ./$(artifact_core-name)/$(artifact_core-name).jar
```

Comment:
Will this ever exist? The core artifact is copied to ./$(artifact_name).jar only.